src/calculator/Calculator.java

```diff
@@ -30,44 +22,40 @@ public static void main(String[] args) {
                     System.out.println("Błąd! Wprowadź poprawną liczbę.");
                     sc.next();
                 }
-                    double a = sc.nextDouble();
+                    double firstUserNumber = sc.nextDouble();
             System.out.println("Podaj drugą liczbę: ");
                 while (!sc.hasNextDouble())
                 {
                     System.out.println("Błąd! Wprowadź poprawną liczbę.");
                     sc.next();
                 }
-                    double b = sc.nextDouble();
+                    double secondUserNumber = sc.nextDouble();
                     sc.nextLine();
             System.out.println("Podaj znak działania, które chcesz wykonać (+ - * / ^): ");
                 String operationSymbol = sc.nextLine();
-
+        
         switch (operationSymbol) {
             case "+":
-                wynik = a+b;
+                result = firstUserNumber+secondUserNumber;
                 break;
             case "-":
-                wynik = a-b;
+                result = firstUserNumber-secondUserNumber;
                 break;
             case "*":
-                wynik = a*b;
+                result = firstUserNumber*secondUserNumber;
                 break;
             case "/":
-                wynik = a/b;
+                result = firstUserNumber/secondUserNumber;
                 break;
             case "^":
-                wynik = Math.pow(a, b);
+                result = Math.pow(firstUserNumber, secondUserNumber);
                 break;
             default:
                 System.out.println("Podałeś zły znak działania!");
                 System.out.println("Spróbuj jeszcze raz");
                 System.exit(0);

```

Comment:
`System.exit()` jest metodą zatrzymującą Java Virtual Machine. Status 0 oznacza normalne zamknięcie, nie potrzebujesz tutaj tego używać. JVM i tak się zamknie po wykonaniu programu.
https://stackoverflow.com/questions/3715967/when-should-we-call-system-exit-in-java